CHANGELOG.md

```diff
@@ -1,5 +1,347 @@
 # Changelog
 
+## v0.72.0-rc.5
+
+### Breaking
+
+- Bump minimum Node JS version to 16 ([afc91de79a](https://github.com/facebook/react-native/commit/afc91de79a8804696f05219214e0e67235d34d77) by [@robhogan](https://github.com/robhogan))
```

Comment:
we need to change this to also mention that we bumped Node to 18 as the used version -> https://github.com/facebook/react-native/pull/35443  (it probably got lost 'cause changelog formatting was weird)